autogluon/utils/tabular/ml/models/lr/lr_model.py

```diff
@@ -140,7 +140,7 @@ def fit(self, X_train, Y_train, X_test=None, Y_test=None, time_limit=None, **kwa
 
         self.model = model.fit(X_train, Y_train)
 
-    def predict_proba(self, X, preprocess=True):
+    def _predict_proba(self, X, preprocess=True):
         X = self.preprocess(X, is_train=False, model_specific_preprocessing=True)
         return super().predict_proba(X, preprocess=False)
```

Comment:
super()._predict_proba(), also make sure this is done everywhere else in predict_proba calls, otherwise normalization will be triggered twice.

Reply:
good catch, fixed it